packages/@react-aria/calendar/test/useCalendar.test.js

```diff
@@ -178,5 +178,45 @@ describe('useCalendar', () => {
       `('should use single $Name', ({defaultValue, rangeBefore, rangeAfter, rel, count, visibleDuration, pageBehavior}) => {
         testPagination(defaultValue, rangeBefore, rangeAfter, rel, count, visibleDuration, pageBehavior);
       });
+
+    it.each`
+    Name | defaultValue | rangeBefore | rangeAfter | rel | count | visibleDuration
+    ${'Week going forward one'} | ${new CalendarDate(2019, 1, 1)} | ${'December 23, 2018 to January 12, 2019'} | ${'January 13 to February 2, 2019'} | ${'Next'} | ${1} | ${{weeks: 3}}
+    ${'Week going forward two'} | ${new CalendarDate(2019, 1, 1)} | ${'December 23, 2018 to January 12, 2019'} | ${'February 3 to 23, 2019'} | ${'Next'} | ${2} | ${{weeks: 3}}
+    ${'Week going backward one'} | ${new CalendarDate(2019, 1, 1)} | ${'December 23, 2018 to January 12, 2019'} | ${'December 2 to 22, 2018'} | ${'Previous'} | ${1} | ${{weeks: 3}}
+    ${'Week going backward two'} | ${new CalendarDate(2019, 1, 1)} | ${'December 23, 2018 to January 12, 2019'} | ${'November 11 to December 1, 2018'} | ${'Previous'} | ${2} | ${{weeks: 3}}
+    `('should use single $Name', ({defaultValue, rangeBefore, rangeAfter, rel, count, visibleDuration, pageBehavior}) => {
+      testPagination(defaultValue, rangeBefore, rangeAfter, rel, count, visibleDuration, pageBehavior);
+    });
+
+    it.each`
+    Name | defaultValue | rangeBefore | rangeAfter | rel | count | visibleDuration | pageBehavior
+    ${'Week going forward one'} | ${new CalendarDate(2019, 1, 1)} | ${'December 23, 2018 to January 12, 2019'} | ${'December 30, 2018 to January 19, 2019'} | ${'Next'} | ${1} | ${{weeks: 3}} | ${'single'}
```

Comment:
this test will have the same name as the one described on like 183, they need to have different names